app/controllers/application_controller.rb

```diff
@@ -50,6 +50,10 @@ def decorated_session
     ).call
   end
 
+  def default_url_options
+    { locale: I18n.locale }
```

Comment:
Are we doing to default `en` to be not there (like the marketing site)? Or are we going to always have a locale in the URL? I think it might be nice to remove it (aka default to english)?